contracts/tests/bridge/test_measure_gas_home_bridge.py

```diff
@@ -93,6 +95,7 @@ def __call__(self, n, fail_ok=False):
     return ConfirmNth()
 
 
+@pytest.mark.skip
```

Comment:
Can you please add a reason right here why we skip them?
And also add a `# TODO`

Reply:
see https://docs.pytest.org/en/latest/skipping.html#skipping-test-functions